pixeltable/catalog/catalog.py

```diff
@@ -639,7 +652,9 @@ def _finalize_pending_ops(self, tbl_id: UUID) -> None:
 
                     if op.needs_xact:
                         tv = self.get_tbl_version(
-                            tbl_id, tbl_version, check_pending_ops=False, validate_initialized=True
+                            TableVersionKey(tbl_id, tbl_version, None),
```

Comment:
None?

Reply:
That is true right now, but won't be in the future.

Add an assertion that checks is_replica.

Reply:
Done. Also added a TODO.